spec/features/refinery/blog/admin/posts_spec.rb

```diff
@@ -152,26 +150,35 @@ module Admin
           end
         end
 
-        context "with multiple users" do
-          let!(:other_guy) { FactoryGirl.create(:authentication_devise_refinery_user, :username => "Other Guy") }
-
-          describe "create blog post with alternate author" do
+        if defined?(Refinery::Authentication::Devise::User)
+          context "with multiple users" do
             before do
-              visit refinery.blog_admin_posts_path
-              click_link "Create new post"
+              allow(Refinery::Blog).to receive(:user_class).and_return(Refinery::Authentication::Devise::User)
+              class Refinery::Blog::Post
+                belongs_to :author, proc { readonly(true) }, :class_name => Refinery::Blog.user_class.to_s, :foreign_key => :user_id
+              end
```

Comment:
This was my doing, and I've got a different/cleaner solution on the way that uses a shared context.  I've issued a PR with it.

The central thing that needs to be accomplished is to refresh the author association to take into account the changed user_class.